src/css/root.int.css

```diff
@@ -0,0 +1,178 @@
+
+	/* Typefaces */
+	--body-font: Inter, Arimo, Charcoal, Geneva, "Helvetica Neue", Helvetica,
+		Arial, sans-serif;
+	--UI-font: PTRootUI, -apple-system, system-ui, BlinkMacSystemFont,
+		"Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantrell, "Helvetica Neue",
+		sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+	--header-font: Inter, Charcoal, "Franklin Gothic Medium", "Franklin Gothic",
+		"ITC Franklin Gothic", "Helvetica Neue", "Helvetica", Arial, sans-serif;
+	--title-font: var(--header-font);
+	--mono-font: Recursive, Consolas, monaco, monospace;
+}
+
+/* Thai Branch */
+:root:lang(th) {
+	/* Header */
+	--header-title: "สถาบัน SCP";
+	--header-subtitle: "ควบคุม, กักกัน, ป้องกัน";
+
+	/* Typefaces */
+	--body-font: Inter, "Kanit", Arimo, -apple-system, system-ui,
+		BlinkMacSystemFont, "Segoe UI", Verdana, Geneva, sans-serif;
+	--UI-font: PTRootUI, "Kanit", "Thonburi", "Leelawadee UI", "Cordia New",
+		Roboto, Noto, "Noto Sans Thai", sans-serif;
+	--header-font: var(--body-font);
+	--title-font: var(--body-font);
+	--mono-font: Recursive, "Kanit", Consolas, monaco, monospace;
+}
+
+/* Traditional Chinese Branch */
+:root:lang(zh) {
+	/* Header */
+	--header-title: "SCP基金會";
+	--header-subtitle: "控制，收容，保護";
+
+	/* Typefaces */
+	--body-font: Inter, "Noto Sans TC", "微軟正黑體", "Microsoft JhengHei",
+		"Microsoft JhengHei UI", “微軟雅黑”, "Microsoft YaHei",
+		"Microsoft YaHei UI", sans-serif;
+	--UI-font: PTRootUI, "方體", "PingFang HK", "方體", "PingFang TC", "黑體",
+		"Heiti TC", "Microsoft JhengHei UI", "Microsoft JhengHei", Roboto, Noto,
+		"Noto Sans CJK TC", sans-serif;
+	--header-font: var(--body-font);
+	--title-font: var(--body-font);
+	--body-font: InRecursiveer, "Noto Sans TC", "微軟正黑體",
+		"Microsoft JhengHei", "Microsoft JhengHei UI", “微軟雅黑”,
+		"Microsoft YaHei", "Microsoft YaHei UI", monospace;
+}
+
+/* Ukrainian Branch */
+:root:lang(uk) {
+	/* Header */
+	--header-title: "SCP Foundation";
+	--header-subtitle: "Ukrainian branch";
+
+	/* Typefaces */
+	--body-font: Inter, Arimo, Charcoal, Geneva, "Helvetica Neue", Helvetica,
+		Arial, sans-serif;
+	--UI-font: PTRootUI, -apple-system, system-ui, BlinkMacSystemFont,
+		"Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantrell, "Helvetica Neue",
+		sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+	--header-font: Inter, Charcoal, "Franklin Gothic Medium", "Franklin Gothic",
+		"ITC Franklin Gothic", "Helvetica Neue", "Helvetica", Arial, sans-serif;
+	--title-font: var(--header-font);
+	--mono-font: Recursive, Consolas, monaco, monospace;
+}
+
+/* French Branch */
+:root:lang(fr) {
+	/* Header */
+	--header-title: "Fondation SCP";
+	--header-subtitle: "Sécuriser, Contenir, Protéger";
+}
+
+/* Spanish Branch */
+:root:lang(es) {
+	/* Header */
+	--header-title: "La Fundación SCP";
+	--header-subtitle: "Rama Hispanohablante";
+}
+
+/* German Branch */
+:root:lang(de) {
+	/* Header */
+	--header-title: "SCP Foundation";
+	--header-subtitle: "auf Deutsch";
+}
```

Comment:
I would, at least in comments, describe all the other country codes and language branches you've identified, to make it easy to add other INT rules.

Reply:
Good call! Did that just now.